src/Workspaces/Core/Portable/Options/EditorConfig/IEditorConfigStorageLocation2.cs

```diff
@@ -24,5 +24,10 @@ internal interface IEditorConfigStorageLocation2 : IEditorConfigStorageLocation
         /// Gets the editorconfig string representation for this storage location.
         /// </summary>
         string GetEditorConfigString(object? value, OptionSet optionSet);
+
+        /// <summary>
+        /// Gets the editorconfig string representation for this storage location. The result does not include the key.
+        /// </summary>
```

Comment:
Probably include an example in the doc comment for this API and the previous API `GetEditorConfigString` to add clarity?

Reply:
Done